tools/coveragerc

```diff
@@ -24,7 +24,6 @@ exclude_lines =
     ^\s*\.\.\.
 
 [run]
-data_file=var/.coverage
```

Comment:
Can you explain this piece of the change a bit more?  Will this adjust what path things get written to in a development environment?

Reply:
Yes, `.coverage` will be written in ~/zulip (or pwd)  but `.coverage` is already in our `.gitignore` file, so this will not bring any change. Anyways, we are not calling the `--no-cov-cleanup` argument in development environment, so this file will be deleted.

Reply:
Right, but I think we want the coverage data to be written to `var/.coverage`, not the root of the project.  Is there a bug in upstream `coverage` related to this as well?

Reply:
This is not related to `coverage` but how `codecov` works when discovering coverage files for uploading. I can't find a setting to specify the location of coverage file when running codecov. (Also google search reveals nothing)

Reply:
Can't we just do `mv` to put things in place for `codecov`?  I think it makes sense to write `coveragerc` for local development (`test-backend --coverage` there) and then fix things up for codecov as needed.